.github/workflows/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'bug'
+assignees: ''
+
+---
+
+**Bug description**
+A clear and concise description of what the bug is. Please do not forget to add:
+- Orion version: X.Y.Z (this can be get with `GET /version` in the API port or using `contextBroker --version`)- 
+MongoDB version X.Y
+- Env variables or CLI parameters
+- Operating System or Docker Image used
+
+**How to reproduce it**
+Steps to reproduce the behavior (as an example)
+1. Create a subscription 
+    ```BASH
+    curl -iX POST 'http://localhost:1026/v2/entities' \
+    -H 'Content-Type: application/json' \
+    -d '
+    {
+        "id": "Bug:007",
+        "type": "bug",
+        "status": {
+            "type": "Text",
+            "value": "fixed"
+        },
+        "created": {
+            "value": "2021-12-01T9:35:00Z",
+            "type": "DateTime"
+        }  
+    }'
+    ```
+2. Check the log '....'
+3. See error in the request response
```

Comment:
```suggestion
2. See error in the request response
3. Check the log '....'
```

More "natural" this way: you would get the error response as result of the curl above before checking the logs.